xwiki-platform-core/xwiki-platform-rendering/xwiki-platform-rendering-macros/xwiki-platform-rendering-macro-code/src/test/resources/macrocode16.test

```diff
@@ -0,0 +1,147 @@
+.runTransformations
+.#-----------------------------------------------------
+.input|xwiki/2.0
+.# Validates that the code macro accurately identifies the language used in the code macro when not specified.
```

Comment:
Same this needs to explain why it's different from the other 2 tests with the same comment